subworkflows/nf-core/bcl_demultiplex/main.nf

```diff
@@ -118,12 +119,14 @@ def generate_fastq_meta(ch_reads, logFile) {
             meta.readgroup = readgroup_from_fastq(fastq)
             meta.readgroup.SM = meta.samplename
         } else {
-            appendToLogFile(
-                "Empty or invalid FASTQ file: ${fastq}",
-                logFile
+            if (params.log_skipped_fastqs) {
```

Comment:
```suggestion
            if (log_skipped_fastqs) {
```

Reply:
For some reason, when I remove `params.` I get `ERROR ~ No such variable: log_skipped_fastqs`